packages/appsync-modelgen-plugin/src/visitors/appsync-visitor.ts

```diff
@@ -726,6 +722,19 @@ export class AppSyncModelVisitor<
         return true;
       });
     });
+
+    Object.values(this.modelMap).forEach(model => {
```

Comment:
N.p. but I'd add a comment addressing why this is a separate loop, rather than in the same one above. Not a blocker though.